tests/test_browser.py

```diff
@@ -74,14 +73,19 @@ def test__request():
     assert data["telephone"] == "555"
     assert data["comments"] == "freezer"
     assert data["size"] == "medium"
-    assert data["topping"] == ["cheese", "onion"]
+    assert data["topping"] == ["bacon", "onion"]
```

Comment:
This looks suspicious...

Reply:
This was simply to fix a flake8 error because the "Extra Cheese" line was too long, and removing "checked" made it shorter. My apologies for not documenting this change.

Reply:
It is now documented in the revised commit message. Thanks for the reminder!